.github/actions/telemetry_check_crwa/action.yaml

```diff
@@ -0,0 +1,5 @@
+name: 'Listen for CRWA telemetry'
+description: 'Builds and executes the CRWA while listening for redirected telemetry'
+runs:
+  using: 'node16'
```

Comment:
Can we make the node version here configurable so we can test on 16 and 18? For reference:

https://github.com/redwoodjs/redwood/blob/9e95de1e9839c2a265e2f81414252835157231c7/.github/actions/set-up-job/action.yml#L6-L8

Reply:
Yeah definitely that was my intention but I must have got distracted getting it working and clearly didn't finish setting the node version correctly 😅. 

I'll update appropriately.